frontend/e2e/protractor.conf.js

```diff
@@ -0,0 +1,31 @@
+// Protractor configuration file, see link for more information
+// https://github.com/angular/protractor/blob/master/lib/config.ts
+
+const { SpecReporter } = require('jasmine-spec-reporter');
+
+exports.config = {
+  allScriptsTimeout: 11000,
+  specs: [
+    './**/*.e2e-spec.ts'
+  ],
+  capabilities: {
+    chromeOptions: {
+      args: ['--headless']
+    },
+    'browserName': 'chrome'
+  },
+  directConnect: true,
+  baseUrl: 'http://localhost:4200/',
+  framework: 'jasmine',
+  jasmineNodeOpts: {
+    showColors: true,
+    defaultTimeoutInterval: 30000,
+    print: function() { }
+  },
+  onPrepare() {
+    require('ts-node').register({
+      project: 'e2e/tsconfig.e2e.json'
```

Comment:
Locally this was looking for ../e2e/e2e/tsconfig.e2e.json, so I had to remove the initial e2e/ here

Reply:
(this was a pycharm issue of how I had their testrunner configured -- once I ran it from the command line I had to add this back in, as expected)